fleet/lib/snmp-exporter-pre/templates/externalsecret-snmp-community.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: snmp-config
+spec:
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: onepassword
+  data:
+    - secretKey: snmpRubinCommunity
+      remoteRef:
+        key: snmp community string
+        property: credential
```

Comment:
There is also a `snmpLsstCommunity` in the config file, so it needs to be exposed as well in the externalSecret

Reply:
Interpolating this value was intentionally omitted as that community string isn't used in Tucson.

Reply:
In another branch, I have work to to template the `externalsecrets` resources with helm values.

Reply:
I've put the multi-site support into https://github.com/lsst-it/k8s-cookbook/pull/285